askama_derive/src/lib.rs

```diff
@@ -11,14 +11,29 @@ use proc_macro2::Span;
 mod config;
 mod generator;
 mod heritage;
+#[cfg(feature = "localization")]
```

Comment:
Maybe we should name the feature `i18n` to match the module name? Seems more consistent.

Reply:
True, that would indeed make sense not only because of the module but also because virtually everywhere i18n is used, will change it once I got time.